.github/workflows/test.yml

```diff
@@ -82,8 +82,6 @@ jobs:
       - name: build (windows-latest)
         if: matrix.os == 'windows-latest'
         run: |-
-          $env:OPENSSL_NO_VENDOR="1"
-          $env:OPENSSL_DIR="$(pwd)\vcpkg\packages\openssl_x64-windows-release"
```

Comment:
If we're no longer using vcpkg openssl, we should maybe delete [the vcpkg](https://github.com/holochain/holochain/blob/65a5059549365cc1615c603a80d1d6a67db4978e/.github/manifest/vcpkg.json#L7) to save CI time.

Reply:
Done https://github.com/holochain/holochain/pull/4798/commits/38fa43045ec529e2c6186f77b7c4aa7763cd3956